STATUS: INLINE FAIL
MODIFIED: com.j256.ormlite.dao.RuntimeExceptionDaoTest.testRollbackThrows
MODIFIED FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
CLEANER: com.j256.ormlite.logger.LoggerFactoryTest.testLogFactoryProperty
CLEANER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
POLLUTER: com.j256.ormlite.logger.LoggerFactoryTest.testSetLogFactory
POLLUTER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 19.578999996185303
==========================
@@ -1068,0 +1068,3 @@
+ {
+     LoggerFactory.setLogBackendFactory(null);
+ }
